input/pagecontent/group-genomics.md

```diff
@@ -1,21 +1,94 @@
+mCODE includes genomics-related data elements needed inform cancer assessment and treatment options. The profiles directly derive from the [HL7 CGWG Clinical Genomics Reporting Implementation Guide](http://hl7.org/fhir/uv/genomics-reporting/index.html) (GRIG) STU2 release. The GRIG provides helpful foundational concepts and guidance for representing different types of genomics reports. mCODE assumes one is familiar with the GRIG content and uses this additional guidance to identify 1) what is clinically actionable and minimal for 
 
-* [GenomicsReport] - Contains results of genomic analyses. Genomic reports vary in complexity and content, as simple as the results for a single discrete variant to complex sequences. This profile inherits from the profile of the same name in CGRIG.
-* [GenomicVariant] - Used to record variants that could be found from tests that broadly analyze genomic regions (e.g.: exome tests) and stores results for any variants that could have been found. The region in which the variant was found could be specified in the RegionStudied attribute of the GenomicsReport profile. This profile inherits from the Variant profile in CGRIG.
-* [GenomicRegionStudied] - Used to record the portion(s) of the genome that was tested for variants. This profile inherits from the RegionStudied profile in CGRIG.
+Three profiles relate to the capture of clinical genomics data:
+
+* [GenomicsReport] - Contains results of genomic analyses. Genomic reports vary in complexity and content, as simple as the results for a single discrete variant to complex sequences. This profile inherits from the profile of the same name in GRIG.
+* [GenomicVariant] - Used to record variants that could be found from tests that broadly analyze genomic regions (e.g.: exome tests) and stores results for any variants that could have been found. The region in which the variant was found could be specified in the RegionStudied attribute of the GenomicsReport profile. This profile derives from the Variant profile in GRIG.
+* [GenomicRegionStudied] - Used to record the portion(s) of the genome that was tested for variants. This profile derives from the RegionStudied profile in GRIG.
+
+Additionally, the additional GRIG profiles are included in the mCODE bundle:
+
+* [Diagnostic Implication](http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition-diagnostic-implication.html)
+* [Therapeutic Implication](http://hl7.org/fhir/uv/genomics-reporting/STU2/StructureDefinition-therapeutic-implication.html)
+* [Microsatellite Instability (MSI)](http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/msi)
+* [Tumor Mutation Burden (TMB)](http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/tmb)
+
```

Comment:
SInce we depend on a specific version of GRIG, links to GRIG should contain the version, explicitly, e.g., http://hl7.org/fhir/uv/genomics-reporting/STU2/StructureDefinition-diagnostic-implication.html

Reply:
good point. Links changed and will be in the next commit.